chunk/chunk_store.go

```diff
@@ -206,6 +208,10 @@ type AWSStore struct {
 	tableName  string
 	bucketName string
 
+	// Around which time to start bucketing indexes by day instead of by hour.
+	// Only the day matters, not the time within the day.
+	dailyBucketsFrom model.Time
```

Comment:
This variable is never initialised!  Which means we'll default to weekly buckets for everything...

We should do two things: copy the config straight into the AWSStore struct, and change the meaning so the time-zero-value means no weekly buckets.

Will backout this PR.